source/gfoidl.Stochastics/Statistics/OutlierDetection/ChauvenetOutlierDetection.cs

```diff
@@ -10,7 +10,7 @@ namespace gfoidl.Stochastics.Statistics
     /// <summary>
     /// Outlier detection based on Chauvenet's criterion.
     /// </summary>
-    public class ChauvenetOutlierDetection : OutlierDetection
+    public sealed class ChauvenetOutlierDetection : OutlierDetection
```

Comment:
This is strictly speaking a breaking change. `sealed` is not necessary perf-wise here anyway, so undo this change.